libs/documentation/src/lib/storybook/utilities/icons/icons.stories.ts

```diff
@@ -0,0 +1,157 @@
+import { IconsUswdsModule } from './icons-uswds/icons-uswds.module';
+
+export default {
+  title: 'Utilities/Icons',
+  decorators: [
+    moduleMetadata({
+      imports: [
+        CommonModule,
+        ReactiveFormsModule,
+        SdsFormlyModule,
+        FormsModule,
+        NoopAnimationsModule,
+        FormlyModule.forRoot(),
+        IconsColoringModule,
+        IconsRotationModule,
+        IconsScalingModule,
+        IconsSkewingModule,
+        IconsStackingModule,
+        IconsBasicModule,
+        IconsBootstrapModule,
+        IconsUswdsModule,
+      ],
+    }),
+  ],
+} as Meta;
+
+export const Coloring: Story = (args) => ({
+  template: '<usa-icon-coloring-demo></usa-icon-coloring-demo>',
+  props: args,
+});
+Coloring.parameters = {
+  controls: {
+    disabled: true,
+    hideNoControlsWarning: true,
+  },
+  actions: { disabled: true },
+  preview: generateConfig('storybook/utilities/icons/icons-coloring', 'IconsColoringModule', 'usa-icon-coloring-demo'),
+  stackblitzLink: generateStackblitzLink('icons-coloring', 'coloring'),
+};
+
+export const Rotating: Story = (args) => ({
+  template: '<usa-icon-rotation-demo></usa-icon-rotation-demo>',
+  props: args,
+});
+Rotating.parameters = {
+  controls: {
+    disabled: true,
+    hideNoControlsWarning: true,
+  },
+  actions: { disabled: true },
+  preview: generateConfig('storybook/utilities/icons/icons-rotation', 'IconsRotationModule', 'usa-icon-rotation-demo'),
+  stackblitzLink: generateStackblitzLink('icons-rotation', 'rotating'),
+};
+
+export const Sizing: Story = (args) => ({
+  template: '<usa-icon-scaling-demo></usa-icon-scaling-demo>',
+  props: args,
+});
+Sizing.parameters = {
+  controls: {
+    disabled: true,
+    hideNoControlsWarning: true,
+  },
+  actions: { disabled: true },
+  preview: generateConfig('storybook/utilities/icons/icons-scaling', 'IconsScalingModule', 'usa-icon-scaling-demo'),
+  stackblitzLink: generateStackblitzLink('icons-scaling', 'Sizing'),
+};
+
+export const Skewing: Story = (args) => ({
+  template: '<usa-icon-skewing-demo></usa-icon-skewing-demo>',
+  props: args,
+});
+Skewing.parameters = {
+  controls: {
+    disabled: true,
+    hideNoControlsWarning: true,
+  },
+  actions: { disabled: true },
+  preview: generateConfig('storybook/utilities/icons/icons-skewing', 'IconsSkewingModule', 'usa-icon-skewing-demo'),
+  stackblitzLink: generateStackblitzLink('icons-skewing', 'skewing'),
+};
+
+export const Stacking: Story = (args) => ({
+  template: '<usa-icon-stacking-demo></usa-icon-stacking-demo>',
+  props: args,
+});
+Stacking.parameters = {
+  controls: {
+    disabled: true,
+    hideNoControlsWarning: true,
+  },
+  actions: { disabled: true },
+  preview: generateConfig('storybook/utilities/icons/icons-stacking', 'IconsStackingModule', 'usa-icon-stacking-demo'),
+  stackblitzLink: generateStackblitzLink('icons-stacking', 'stacking'),
+};
+
+export const Custom: Story = (args) => ({
```

Comment:
Can the name of this and the bootstrap and uswds examples be updated to include Icons? So Custom Icons, Bootstrap Icons, USWDS Icons? I feel that its more clear that the list of icons would be under examples with those names rather than just Custom, or USWDS.